catalog/harvest/catalog-harvest-webdav/src/main/java/org/codice/ddf/catalog/harvest/webdav/WebDavHarvester.java

```diff
@@ -54,7 +51,7 @@
 
   private final Sardine sardine = SardineFactory.begin();
 
-  private Map<String, Serializable> attributeOverrides = new HashMap<>();
+  private Map<String, List<String>> attributeOverrides = new HashMap<>();
```

Comment:
✏️ Something I thought of right now is when these Harvesters eventually support ingesting the content into the content store, we'll want to use the content frameworks method of attribute overrides.